forwarder/routes.go

```diff
@@ -67,12 +67,13 @@ type Router struct {
 	// Owner is the routers owner
 	Owner common.Address
 	// JoinFilter is the filter of devices that are allowed to join the network this router is part of
-	joinFilter xorfilter.Xor8
+	joinFilter      *xorfilter.Xor8
+	joinFilterMutex sync.RWMutex
```

Comment:
There is a convention that the mutex is placed above the fields that it is guarding.